src/corecel/sys/Stream.cc

```diff
@@ -17,14 +17,19 @@
 
 namespace celeritas
 {
+#if CELER_USE_DEVICE
```

Comment:
Maybe add some information here about which gcc version made us make this change (to later somebody might date remove it).